framework/content/ecl-typography/ecl-typography-headings/_headings.scss

```diff
@@ -1,46 +1,78 @@
 // Headers
-.ecl-h1,
-.ecl-h2,
-.ecl-h3,
-.ecl-h4,
-.ecl-h5,
-.ecl-h6 {
+.ecl-heading {
   margin: 0;
-  padding: 0.5em 0;
+  padding: 0;
 }
 
-.ecl-h1 {
-  color: $black;
+.ecl-heading--h1 {
+  color: #000;
```

Comment:
Should use a variable (also valid for every following rules using a value directly)